src/assistant_library_with_local_commands_demo.py

```diff
@@ -90,7 +91,8 @@ def process_event(assistant, event):
 
 def main():
     credentials = aiy.assistant.auth_helpers.get_assistant_credentials()
-    with Assistant(credentials) as assistant:
+    device_id, model_id = aiy.assistant.device_helpers.get_ids(credentials)
+    with Assistant(credentials, model_id) as assistant:
```

Comment:
shouldn't you pass the `device_id` there?

Reply:
Not sure what you mean - this is using the library like the other examples. I'll use the device_id once I switch to v1alpha2